tests/utils/tests/test_schemapi.py

```diff
@@ -380,6 +383,113 @@ def test_schema_validation_error():
     assert the_err.message in message
 
 
+def chart_example_layer():
+    points = (
+        alt.Chart(data.cars.url)
+        .mark_point()
+        .encode(
+            x="Horsepower:Q",
+            y="Miles_per_Gallon:Q",
+        )
+    )
+    return (points & points).properties(width=400)
+
+
+def chart_example_hconcat():
+    source = data.cars()
+    points = (
+        alt.Chart(source)
+        .mark_point()
+        .encode(
+            x="Horsepower",
+            y="Miles_per_Gallon",
+        )
+    )
+
+    text = (
+        alt.Chart(source)
+        .mark_text(align="right")
+        .encode(alt.Text("Horsepower:N", title=dict(text="Horsepower", align="right")))
+    )
+
+    return points | text
+
+
+def chart_example_invalid_channel_and_condition():
+    selection = alt.selection_point()
+    return (
+        alt.Chart(data.barley())
+        .mark_circle()
+        .add_params(selection)
+        .encode(
+            color=alt.condition(selection, alt.value("red"), alt.value("green")),
+            invalidChannel=None,
+        )
+    )
+
+
+@pytest.mark.parametrize(
+    "chart_func,expected_error_message",
```

Comment:
Without space is the style pytest uses in [their documentation](https://docs.pytest.org/en/6.2.x/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions). I think this is the first occurence in Altair tests with multiple arguments. Both works so I don't mind to adopt a style with space :)

Reply:
There are actually a few places that use this style in Altair already and they seem to use a space. For example:

https://github.com/altair-viz/altair/blob/8a8642b2e7eeee3b914850a8f7aacd53335302d9/tools/schemapi/tests/test_utils.py#L36-L40

and 

https://github.com/altair-viz/altair/blob/97ff1ebc3a485be1c00a34e5d19801eca5287e6f/tests/vegalite/v5/tests/test_api.py#L179-L183

Personally I don't really mind which style we use, but it would be nice to be consistent (although honestly I think this is very minor too so let me know what you prefer and we can merge this PR afterwards).

Reply:
Ah sorry, not sure how I missed those. I like the consistency as well so I committed your code suggestion. Good to go from my side, thanks for the review @joelostblom!